src/stream/src/executor/managed_state/aggregation/extreme.rs

```diff
@@ -124,23 +130,22 @@ where
     /// always be retained when flushing the managed state. Otherwise, we will only retain n entries
     /// after each flush.
     pub async fn new(
-        keyspace: Keyspace<S>,
+        _keyspace: Keyspace<S>,
```

Comment:
We may remove the generic for the struct and add it to every functions accepting the state table. So that there's no need for phantom data as well.